src/common/marlin_client.c

```diff
@@ -732,17 +666,17 @@ uint32_t _wait_ack_from_server(uint8_t client_id) {
 
 // process message on client side (set flags, update vars etc.)
 void _process_client_message(marlin_client_t *client, variant8_t msg) {
-    char var_str[16];
+    char var_str[128];
```

Comment:
Magical constant? What is the reason for making it bigger? Did you check stack usage?

Reply:
this buffer is just for trace I will make define and limit output with snprintf.

Reply:
commit aa4f145